### PR TITLE
#168451420 build reject session endpoint

### PR DESCRIPTION
**Pull request by bbaime98**
#### What does this PR do?
- add reject session endpoint 
- add reject session endpoint tests
#### Description of Task to be completed?
- add reject session endpoint
- add reject session test
#### How should this be manually tested?
 -clone repo
- test patch('/api/v1/sessions/:id/reject) with postman
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- #168451420
#### Screenshots (if appropriate)
image

### CODE REVIEW

File: server/controllers/SessionController.js [hunk 1]
```diff
@@ -79,48 +79,53 @@ static async createSession(req, res) {
         })
        
 };
-    static rejectSession(req, res) {
+    // static rejectSession(req, res) {
```
Comment (ekumamait) [inline server/controllers/SessionController.js]: does this commented code have any business here

File: server/tests/sessionTest.js [hunk 2]
```diff
@@ -76,9 +76,17 @@ describe('sessions test',()=>{
...
             done();
         })
-    })
-
-
+    });
+    it('should return session rejected',(done)=>{
 
+        chai.request(app)
+        .patch('/api/v1/sessions/8/reject')
```
Comment (ekumamait) [inline server/tests/sessionTest.js:83]: the versioning should have changed to `v2`

**Review by ekumamait — COMMENTED** [review]
Commented code should not be pushed to github

File: server/tests/sessionTest.js (hunk 2 above)
Comment (bbaime98) [reply server/tests/sessionTest.js:83]: > the versioning should have changed to `v2`

I have used v1 on all routes, the difference is on the last naming of the route

**Pull request merged**